src/model/liveposition.js

```diff
@@ -123,12 +123,16 @@ export default class LivePosition extends Position {
  */
 function bindWithDocument() {
 	/*jshint validthis: true */
+	// Operation types handled by LivePosition (these are operations that change model tree structure).
+	const supportedTypes = new Set( [ 'insert', 'move', 'remove', 'reinsert' ] );
```

Comment:
It might look better (if it is static), but I don't want to create new `Set` or `Array` each time it is called so it would have to be something like:

```javascript
static get supportedTypes() {
    return supportedTypes;
};
// ... after class declaration
const supportedTypes = new Set( ... );
```

I think we are fine with how it looks like now.